### PR TITLE
Refs [TASK] [34079] Init project

### PR DESCRIPTION
**Pull request by Hoang-Kelvin**
## Related Tickets
- [#Task 34079:  Init project]({https://edu-redmine.sun-asterisk.vn/issues/xxxx})
## WHAT
- Initialize empty project
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/values/values-night/themes.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (datdt-1935) [inline app/src/main/res/values/values-night/themes.xml]: file này có thừa không nhỉ?

File: app/src/main/res/values/values-v28/styles.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (datdt-1935) [inline app/src/main/res/values/values-v28/styles.xml]: cả file này nữa

File: app/src/main/res/values/values-night/themes.xml (hunk 1 above)
Comment (Hoang-Kelvin) [reply app/src/main/res/values/values-night/themes.xml]: thêm file này vào để sử dụng thuộc tính tools:targetApi thay cho việc tạo ra file resources mới

File: app/src/main/res/values/values-v28/styles.xml (hunk 2 above)
Comment (Hoang-Kelvin) [reply app/src/main/res/values/values-v28/styles.xml]: thêm file này để sử dụng các kí tự đặc biệt, nhưng mà hiện tại chưa dùng nên đang bị thừa

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**